core/src/main/java/nl/nn/adapterframework/filesystem/FileSystemUtils.java

```diff
@@ -97,8 +98,8 @@
 	}
 
 	public static <F> F moveFile(IBasicFileSystem<F> fileSystem, F file, String destinationFolder, boolean overwrite, int numOfBackups, boolean createFolders) throws FileSystemException {
-		checkSource(fileSystem, file, "move");
-		prepareDestination(fileSystem, file, destinationFolder, overwrite, numOfBackups, createFolders, "move");
+		checkSource(fileSystem, file, FileSystemAction.MOVE.getLabel());
```

Comment:
in checkSource you're just validating if the action is allowed (if I'm not mistaken) you could do `xx.getLabel()` there.
That way on line 91 you'd also pass the enum, instead of a String